crates/paimon/src/spec/snapshot.rs

```diff
@@ -134,4 +144,38 @@ impl Snapshot {
     pub fn statistics(&self) -> Option<&str> {
         self.statistics.as_deref()
     }
+
+    /// Get the commit time of this snapshot.
+    #[inline]
+    pub fn time_millis(&self) -> u64 {
+        self.time_millis
+    }
+
+    /// Get the commit identifier of this snapshot.
+    #[inline]
+    pub fn commit_identifier(&self) -> i64 {
+        self.commit_identifier
+    }
+
+    /// Get the commit kind of this snapshot.
+    #[inline]
+    pub fn commit_kind(&self) -> CommitKind {
+        self.commit_kind
+    }
+}
+
+/// Type of changes in this snapshot.
+///
+/// Impl Reference: <https://github.com/apache/paimon/blob/release-0.8.2/paimon-core/src/main/java/org/apache/paimon/Snapshot.java#L506>.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub enum CommitKind {
```

Comment:
@Aitozi We do not need this enum, you can refer #11.

Reply:
Hi @QuakeWang I think that just means `log_offsets` ?